assets/js/pages/project/details/members.vue

```diff
@@ -21,7 +22,45 @@
                         </v-col>
                     </v-row>
                 </td>
-                <td>{{ member.role }}</td>
+                <td class="members__col-role">
+                    <template v-if="canChangeMemberRole(member)">
+                        <v-select
+                                @input="value => changeRole(member.username, value)"
+                                :items="availableRoles"
+                                :value="member.role"
+                        ></v-select>
+                    </template>
+                    <template v-else>
+                        {{ member.role|roleLabel }}
+                    </template>
+                </td>
+                <td class="members__col-more">
+                    <v-menu>
+                        <template v-slot:activator="{ on }">
+                            <v-btn icon v-on="on">
+                                <v-icon>more_vert</v-icon>
+                            </v-btn>
+                        </template>
+                        <v-list>
+                            <v-list-item link @click.prevent>
+                                <v-list-item-icon>
+                                    <v-icon>mdi-face</v-icon>
```

Comment:
I am not completly sure, but maybe this icon should be aligned with top right dropdown "profile link" icon.

Reply:
I had to use that face somewhere :D. Yes and no, in the top right icon shows editing profile. We can change this later.